eidpauth/src/main/java/uk/ac/ucl/eidp/auth/AuthService.java

```diff
@@ -4,8 +4,10 @@
 
 import java.util.Set;
 import java.util.UUID;
+import java.util.stream.Collectors;
 import javax.ejb.EJB;
 import javax.ejb.Stateless;
+import uk.ac.ucl.eidp.auth.model.RoleE;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'uk\.ac\.ucl\.eidp\.auth\.model\.RoleE' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.